doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -60,54 +60,54 @@ The issues in this section are related to :ref:`protocols`.
 Amazon Sidewalk
 ===============
 
```

Comment:
```suggestion

.. rst-class:: v2-6-0

KRKNWK-11630: BUS fault on FSK during the FACTORY_RESET call.
  Bluetooth LE link is deinitialized, however, a race condition might occur where an event added by the Bletooth LE link is not removed. 
  Acting on the event without a valid Bluetooth LE handle leads to dereference of ``NULL``.
  To avoid the error, you need to check for ``NULL`` before the event is processed in the event queue.
  The issue requires a fix in the Sidewalk stack.
 
  **Workaround:** Perform a factory reset as the error is recoverable.
 
.. rst-class:: v2-6-0

KRKNWK-11630: Advertising fails to start (``err -12``) upon registration.
  The Sidewalk end device is trying to restart advertising, for example, on time sync lost. 
  This happens even when the device is connected through Bluetooth LE, leading to an error.
  The Bluetooth LE connection management requires a fix in the Sidewalk stack.
 
  **Workaround:** The issue fixes upon automatic restart of the Bluetooth LE advertising.
 
.. rst-class:: v2-6-0
 
KRKNWK-17653: Assertion fails during the stress test (pressing different buttons).
  There is a corner case in dynamic heap allocation which leads to a double free error.

  **Workaround:**  The device resets after assertion and functions normally.

```

Reply:
Done